src/elements/content-sidebar/Sidebar.js

```diff
@@ -236,6 +326,8 @@ class Sidebar extends React.Component<Props, State> {
             'bcs-is-open': isOpen,
         });
 
+        const { currentUser, currentUserError } = this.state;
```

Comment:
Why do we need to fetch and get currentUser from state rather than props?

Reply:
`ActivitySidebar` needs a `currentUser` to post comments. When `ContentPreview` is used in the webapp, the `currentUser` is provided based on who is logged in verse using the token and file info to assume the `currentUser`.  So if the `currentUser` is retrieved separately and passed via props then its not fetched at all.